WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -581,4 +789,58 @@ public void onPostUploaded(OnPostUploaded event) {
             }
         }
     }
+
+    @Override public void onPositiveClicked(@NonNull String instanceTag) {
```

Comment:
Even thought Android Studio automatically adds it in the same line, the `@Override` annotation for the `onPositiveClicked`, `onNegativeClicked`, `onSettingsSaved`, `onSaveError`, `onFetchError`, `onSettingsUpdated`, and `onCredentialsValidated` methods should probably be on a separate line for code style consistency.